presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -181,9 +182,10 @@ void setAggregationSpillConfig(
   queryConfigs.emplace(core::QueryConfig::kAggregationSpillEnabled, "true");
 }
 
-class TaskManagerTest : public testing::Test {
+class TaskManagerTest : public exec::test::OperatorTestBase {
```

Comment:
Maybe not for this PR. But since almost all test are extending from OperatorTestBase, we might want to change a name of that class.